pages/common/supervisorctl.md

```diff
@@ -0,0 +1,24 @@
+# supervisorctl
+
+> Supervisor is a client/server system that allows its users to control a number of processes on UNIX-like operating systems.
+> Supervisorctl is the command-line client piece of the supervisor which provides a shell-like interface.
+
+- Start/stop/restart a process:
+
+`supervisorctl {{start|stop|restart}} {{process name}}`
+
+- Start/stop/restart all processes in a group:
+
+`supervisorctl {{start|stop|restart}} {{group name}}:*`
+
+- Show last N **bytes** of process stderr:
+
+`supervisorctl tail -{{N}} {{process name}} stderr`
+
+- Keep showing named process stdout until ctrl-c:
```

Comment:
"named process stdout" was token from supervisor's help, I found it reads wired too, I will update this.

ctrl-c part was token from tldr/common/tail.md, I wanted keep it consistent, it could be removed though.